tool/src/org/antlr/v4/tool/ErrorType.java

```diff
@@ -956,6 +956,8 @@
 	 */
 	CHANNELS_BLOCK_IN_COMBINED_GRAMMAR(164, "custom channels are not supported in combined grammars", ErrorSeverity.ERROR),
 
+	NONCONFORMING_LR_RULE(165, "rule <arg> is left recursive but doesn't conform to a pattern ANTLR can handle", ErrorSeverity.ERROR),
```

Comment:
This error number conflicts with the error numbers in #697. Since those error codes shipped in the 4.4 release of the optimized fork, it would be helpful if `NONCONFORMING_LR_RULE` was renumbered as error 169 (leaving space for 165 through 168 to be added to this project when #697 is ready).